ament_cmake_test_isolated/cmake/run_test_isolated.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+
+import ament_cmake_test
+import domain_coordinator
+
+
+if __name__ == '__main__':
+    # If ROS_DOMAIN_ID is already set, respect that domain ID and use it.
+    # If ROS_DOMAIN_ID is not set, pick a ROS_DOMAIN_ID that's not being used
+    # by another run_test_isolated process.
```

Comment:
It might be nice to have this kind of documentation copied or moved up to the cmake function's documentation, as users are far more likely to read the doc blocks for the cmake functions they are calling than to find this comment here. And I think this insight into how it works is worth noting in a place that the user is more likely to see it.

Reply:
I agree